inspirehep/dojson/geo_helpers.py

```diff
@@ -0,0 +1,420 @@
+    'ALASKA': 'US-AK',
+    'ARIZONA': 'US-AZ',
+    'ARKANSAS': 'US-AR',
+    'CALIFORNIA': 'US-CA',
+    'COLORADO': 'US-CO',
+    'CONNECTICUT': 'US-CT',
+    'DELAWARE': 'US-DE',
+    'DISTRICT OF COLUMBIA': 'US-DC',
+    'FLORIDA': 'US-FL',
+    'GEORGIA': 'US-GA',
+    'HAWAII': 'US-HI',
+    'IDAHO': 'US-ID',
+    'ILLINOIS': 'US-IL',
+    'INDIANA': 'US-IN',
+    'IOWA': 'US-IA',
+    'KANSAS': 'US-KS',
+    'KENTUCKY': 'US-KY',
+    'LOUISIANA': 'US-LA',
+    'MAINE': 'US-ME',
+    'MARYLAND': 'US-MD',
+    'MASSACHUSETTS': 'US-MA',
+    'MICHIGAN': 'US-MI',
+    'MINNESOTA': 'US-MN',
+    'MISSISSIPPI': 'US-MS',
+    'MISSOURI': 'US-MO',
+    'MONTANA': 'US-MT',
+    'NEBRASKA': 'US-NE',
+    'NEVADA': 'US-NV',
+    'NEW HAMPSHIRE': 'US-NH',
+    'NEW JERSEY': 'US-NJ',
+    'NEW MEXICO': 'US-NM',
+    'NEW YORK': 'US-NY',
+    'NORTH CAROLINA': 'US-NC',
+    'NORTH DAKOTA': 'US-ND',
+    'OHIO': 'US-OH',
+    'OKLAHOMA': 'US-OK',
+    'OREGON': 'US-OR',
+    'PENNSYLVANIA': 'US-PA',
+    'RHODE ISLAND': 'US-RI',
+    'SOUTH CAROLINA': 'US-SC',
+    'SOUTH DAKOTA': 'US-SD',
+    'TENNESSEE': 'US-TN',
+    'TEXAS': 'US-TX',
+    'UTAH': 'US-UT',
+    'VERMONT': 'US-VT',
+    'VIRGINIA': 'US-VA',
+    'WASHINGTON': 'US-WA',
+    'WEST VIRGINIA': 'US-WV',
+    'WISCONSIN': 'US-WI',
+    'WYOMING': 'US-WY'
+}
+
+us_states_alternative_spellings = {
+    'US-AL': ['AL', 'ALA'],
+    'US-AK': ['AK'],
+    'US-AZ': ['AZ', 'ARIZ'],
+    'US-AR': ['AR'],
+    'US-CA': ['CA', 'CALIFORNIA', 'CALIF', 'CALF'],
+    'US-CO': ['CO', 'COLO'],
+    'US-CT': ['CT', 'CONNETICUT'],
+    'US-DC': ['DC'],
+    'US-DE': ['DE', 'DEL'],
+    'US-FL': ['FL'],
+    'US-GA': ['GA'],
+    'US-HI': ['HI'],
+    'US-ID': ['ID'],
+    'US-IL': ['IL', 'ILL', 'ILLINOIS'],
+    'US-IN': ['IN'],
+    'US-IA': ['IA'],
+    'US-KS': ['KS'],
+    'US-KY': ['KY'],
+    'US-LA': ['LA'],
+    'US-ME': ['ME'],
+    'US-MD': ['MD'],
+    'US-MA': ['MA', 'MASS'],
+    'US-MI': ['MI', 'MICH'],
+    'US-MN': ['MN'],
+    'US-MS': ['MS'],
+    'US-MO': ['MO'],
+    'US-MT': ['MT'],
+    'US-NE': ['NE', 'NEV'],
+    'US-NV': ['NV'],
+    'US-NH': ['NH'],
+    'US-NJ': ['NJ'],
+    'US-NM': ['NM', 'NMEX', 'NEW MEXIKO', 'NEW MEX', 'N MEX', 'NEW MEXICO'],
+    'US-NY': ['NY'],
+    'US-NC': ['NC'],
+    'US-ND': ['ND'],
+    'US-OH': ['OH'],
+    'US-OK': ['OK'],
+    'US-OR': ['OR'],
+    'US-PA': ['PA'],
+    'US-RI': ['RI'],
+    'US-SC': ['SC'],
+    'US-SD': ['SD'],
+    'US-TN': ['TN', 'TENN'],
+    'US-TX': ['TX', 'TEX'],
+    'US-UT': ['UT'],
+    'US-VT': ['VT'],
+    'US-VA': ['VA'],
+    'US-WA': ['WA'],
+    'US-WV': ['WV'],
+    'US-WI': ['WI', 'WIS', 'WISC'],
+    'US-WY': ['WY'],
+}
+
+south_korean_cities = ['SEOUL', 'DAEJON', 'DAEJEON', 'MT SORAK', 'POHANG',
+                       'JEJU ISLAND', 'CHEJU ISLAND', 'GYEONGJU', 'BUSAN',
+                       'DAEGU', 'GYEONGIU', 'PUSAN', 'YONGPYONG',
+                       'PHOENIX PARK', 'CHEJU ISLAND']
```

Comment:
Agreed, it is a weird file. As I said above it should be replaced with GoogleGeocoding although AFAIK we do not use GoogleGeo yet. For now this extracts `country` and `U.S. state` from vast majority of production's records (the way it does that is silly but was quick to implement) - and those are the information needed for Neo4J. If we replace it with something smarter, I'll be more than happy! For now I'd rather have some usable data than none.

The data come more or less from [ISO_3166-1 alpha 2 standard](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) if you like we can store it the other way :smile: